evap/results/templates/results_index_evaluation.html

```diff
@@ -18,9 +18,7 @@
                 {% endfor %}
                 <span class="badge badge-secondary badge-course-type">{{ evaluation.course.type }}</span>
             {% endif %}
-            {% if evaluation.is_single_result %}
-                <span class="badge badge-success">{% trans 'Single result' %}</span>
-            {% endif %}
+            {% include 'evaluation_badges.html' with mode='result' %}
```

Comment:
As I understand the code, you want to hide degrees and course type on sub-evaluations.
Could you use the “normal” mode on “fat” evaluations and your “results” mode for sub-evaluations?
I think it would also be fine to have `.badge-degree` and `.badge-course-type` in `evaluation_badges.html`